portal/src/graphql/portal/AuthenticatorConfigurationScreen.tsx

```diff
@@ -333,6 +334,15 @@ const AuthenticationAuthenticatorSettingsContent: React.FC<AuthenticationAuthent
       [state.mfaMode]
     );
 
+    const isPhoneLoginIdDisabled = useMemo(
+      () =>
+        effectiveConfig.identity?.login_id?.keys?.find(
+          (t) => t.type === "phone"
+        )?.modify_disabled ?? true,
```

Comment:
No need to check modify_disabled. If an item of type `phone` exists in the array, phone login ID is enabled.